.changeset/dull-spies-love.md

```diff
@@ -0,0 +1,18 @@
+---
+'@talend/react-faceted-search-query-client': major
+'@talend/react-faceted-search': major
+---
+
+[BREAKING] Move the function createTqlQuery to a dedicated package (@talend/react-faceted-search-query-client).
+
+You will need to adapt your code to use this lib
+
+before
+```
+import { createTqlQuery } from '@talend/react-faceted-search/lib/queryClient';
```

Comment:
```suggestion
import { createTqlQuery } from '@talend/react-faceted-search/lib/queryClient';
import { queryClient } from '@talend/react-faceted-search';
```

Reply:
I checked the usage in projects and it's only used with /lib...
I suppose it's because the export of faceted-search is a default one
So in this case teams had to import the default and then extract queryClient

Reply:
Yes but it was possible to import it like this so we have to document that possible usage